### DISCUSSION TITLE
Removing Bookmarks / Current Status links

### DISCUSSION THREAD

**Discussion by StillCantExitVim** [sni/thruk#1131] [Q&A] [answered]
Not sure if this is the right place to ask, but it seems more active than Stackoverflow for Thruk questions.
I'm running nagios 4 with thruk 2.44.3 installed from labs_consol_stable rpm on rhel 8. 

I've used the default thrukadmin user to change filter variables on the "Problems" (All Unhandled Problems) link from the menu bar and used the "Add a bookmark to this page" button to save it under "Current Status". It appears under the search bar as expected. However now I'm wondering if it's possible to edit/remove this bookmark. When I click the bookmark I only get the option to add another one, not to edit or remove the bookmark. I do not see any references to the newly created in either menu_local.conf or /usr/share/thruk/menu.conf

How do I edit or remove the newly created bookmark?

**Comment by sni** [comment, marked as answer]
<img width="600" alt="Screenshot 2021-10-01 at 13 51 17" src="https://user-images.githubusercontent.com/150844/135615672-e9b64d0b-5673-4600-91d3-5bb627565e86.png">

There is a "remove bookmarks" link in that popup. Does that work for you?

**Reply by StillCantExitVim** [reply]
Ah, I completely missed that part of the popup, yes that works ,thank you!
